frontend/src/Constants/ComponentText.ts

```diff
@@ -2,6 +2,8 @@ const ComponentText = {
   NEXT: 'Next',
   ADD: 'Add',
   CANCEL: 'Cancel',
+  NEWFIELD: 'Add Field',
```

Comment:
To keep consistent in Constants naming, I'd name thid 'ADDFIELD'

Reply:
Seeing as this might expand, `ADD_FIELD` might be more appropriate :)